packages/invoices/src/OutboundShipment/DetailView/modals/OutboundLineEditTable.tsx

```diff
@@ -65,9 +66,14 @@ const BatchesRow: React.FC<BatchesRowProps> = ({
       <BasicCell align="right">{batch.totalNumberOfPacks}</BasicCell>
       <BasicCell>{batch.batch}</BasicCell>
       <BasicCell
-        sx={{ color: isAlmostExpired(expiryDate) ? 'error.main' : 'inherit' }}
+        sx={{
+          color:
+            batch.expiryDate && isAlmostExpired(new Date(batch.expiryDate))
+              ? 'error.main'
+              : 'inherit',
+        }}
       >
-        {d(expiryDate)}
+        {expiryDate}
```

Comment:
This was fixing an issue where expiry dates were undefined. Looking at the API it seems as though expiry dates are not optional and so this may have been a problem specifically for the mock api. However in practise I believe expiry dates are generally optional for batches. At least in current mSupply?